### ISSUE TITLE
AppleSignIn is not validating

### ISSUE THREAD

**Issue by robinhoc** [DelphiWorlds/Kastri#30]
Problem with AppleSignIn Apple.

**Description**
I downloaded the kastri fonts, using the demo itself I always get the login failure message, I already enabled the Apple login option in the app ID, the certificate shows that it is enabled, it opens the app all right, it shows the login button but when clicking on login failure.

** Error messages **
Login failed.

** Please indicate **
Delphi 10.4.1 with patch for ios 14.
Iphone XS Max IOS 14.2
MacOS Catalina.
Xocode 12
Added the authentication framework to the SDK.

** Steps to reproduce **
When installing the demo on the phone and trying to login already gives the error message.

**Comment by DelphiWorlds** [comment]
Very odd. Just tested it again myself and it works fine, here. 
Delphi 10.4.1, iOS 14.2 SDK, Xcode 12.2
Steps:
1. Compile/run
2. Tap "Sign in with Apple" button
3. Tap "Continue"
4. Allow device to use Face ID to verify
Authorization succeeds

**Comment by robinhoc** [comment]
Really strange, here I am using the demo that is in the project folder.

I did the following procedure:
In the apple developer I went to identifiers, I have 2, in the 2 I enabled the option Sign In with Apple, when editing this selected Enable as a primary App ID and in server I put a link to my site.

I compiled the demo in delphi 10.4.1 already with the patch of the shipper, on an iphone XS Max IOS 14.2, the SDK in delphi and the 14, xcode 12.

It opens correctly, shows the button but always falls in the option login failure, what can it be to always fall in this validation?
